gensim/models/utils_any2vec.py

```diff
@@ -335,7 +335,7 @@ def _load_word2vec_format(cls, fname, fvocab=None, binary=False, encoding='utf8'
         counts = {}
         with utils.open(fvocab, 'rb') as fin:
             for line in fin:
-                word, count = utils.to_unicode(line, errors=unicode_errors).strip().split()
+                word, count = utils.to_unicode(line, encoding=encoding, errors=unicode_errors).split(" ")
```

Comment:
What's up with this sneaky change to `" "`? And dropping `strip`?

This changes the semantics considerably, not just "Added unicode_errors arg".

Reply:
I had to focus on the change I proposed. I'll fix this soon.

Reply:
@piskvorky I've reverted except "add unicode_errors arg" and checked the test codes work.

Reply:
OK. Btw I'm not saying these changes were necessarily bad – I'm saying they were under-documented, with unclear (to me) motivation and implications.

Reply:
Thanks for your review. The change you pointed out was not well considered. I understood what you said, but I thought I should focus on fixing the reported issue, for now, the other changes should be reported and PRed by another issue and PR. Thanks!